lib/MetaCPAN/Web/Controller/Permission.pm

```diff
@@ -15,6 +16,16 @@ sub distribution : Local Args(1) {
     my ( $self, $c, $distribution ) = @_;
 
     $c->forward( 'get', $c, [ 'distribution', $distribution ] );
+
+    my $modules = $c->stash->{permission};
+    my %num_modules_of;
+    for (@$modules) {
+        ++$_ for @num_modules_of{ $_->{owner}, @{ $_->{co_maintainers} } };
+    }
+    my @releaser
+        = sort grep { $num_modules_of{$_} == @$modules } keys %num_modules_of;
+
```

Comment:
Could we make the above code less succinct?  It's not immediately obvious what is going on in lines 22-26.  ;)

Reply:
How? @haarg already reviewed it twice and asked me to make it clearer the first time, so I moved some things around slightly and gave the variables better names. Having done that, I’m not sure how I can make this algorithm any clearer, and I couldn’t think of any algorithm that will be easier to understand. Do you have any suggestion?